ush/rocoto/workflow_utils.py

```diff
@@ -275,27 +275,33 @@ def get_resources(machine, cfg, task, reservation, cdump='gdas'):
 
     scheduler = get_scheduler(machine)
 
-    if cdump in ['gfs'] and 'wtime_%s_gfs' % task in cfg.keys():
-        wtimestr = cfg['wtime_%s_gfs' % task]
+    if cdump in ['gfs'] and f'wtime_{task}_gfs' in cfg.keys():
+        wtimestr = cfg[f'wtime_{task}_gfs']
     else:
-        wtimestr = cfg['wtime_%s' % task]
+        wtimestr = cfg[f'wtime_{task}']
 
     ltask = 'eobs' if task in ['eomg'] else task
 
-    memory = cfg.get('memory_%s' % ltask, None)
+    memory = cfg.get(f'memory_{ltask}', None)
 
-    if cdump in ['gfs'] and 'npe_%s_gfs' % task in cfg.keys():
-        tasks = cfg['npe_%s_gfs' % ltask]
+    if cdump in ['gfs'] and f'npe_{task}_gfs' in cfg.keys():
+        tasks = cfg[f'npe_{ltask}_gfs']
     else:
-        tasks = cfg['npe_%s' % ltask]
+        try:
+            tasks = cfg[f'npe_{ltask}']
+        except KeyError:
+            tasks = cfg["',)npe_waveawipsgridded"]
 
-    if cdump in ['gfs'] and 'npe_node_%s_gfs' % task in cfg.keys():
-        ppn = cfg['npe_node_%s_gfs' % ltask]
-    else:
-        ppn = cfg['npe_node_%s' % ltask]
+    if cdump in ['gfs'] and f'npe_node_{task}_gfs' in cfg.keys():
+        ppn = cfg[f'npe_node_{ltask}_gfs']
+    else:        
+        ppn = cfg[f'npe_node_{ltask}']
 
     if machine in [ 'WCOSS_DELL_P3', 'HERA', 'ORION']:
-        threads = cfg['nth_%s' % ltask]
+        try:
+            threads = cfg[f'nth_{ltask}']
+        except KeyError:
+            threads = cfg["',)nth_epos"]
```

Comment:
@BrianCurtis-NOAA 
Same comment as above.
I don't think we have a key `',)nth_epos` in the `cfg` dictionary.